tests/foreman/api/test_contentview.py

```diff
@@ -1722,3 +1722,33 @@ def test_positive_publish_promote_with_RH_ostree_and_other(
         content_view.publish()
         promote(content_view.read().version[0], module_lce.id)
         assert len(content_view.read().version[0].read().environment) == 2
+
+    @pytest.mark.stubbed
+    @pytest.mark.tier1
+    def test_positive_check_pulp3_is_installed(self):
```

Comment:
Do we need separate manual test cases for this? Is there no easy automation path for checking pulp3 processes or RPMs?

There are no test steps, so I'm not sure what your intent is for validating that pulp3 is active and pulp2 is not.

For a manual case, this just doubles the number of test results you need to enter for what is essentially two steps of one test case.

Reply:
The test cases can be condensed and I can add test steps as necessary for manual processes.  For now, these are just stubs that I can revisit to add automation later or can leave in draft state until automation is added.  One verification is via UI that pulp3 is installed.  Another is ensuring that you cannot connect to the pulp2 db.

Reply:
Hi, I added a check for presence of Pulp3 file system, which would not exists if Pulp3 had not been installed: https://github.com/SatelliteQE/robottelo/blob/master/tests/foreman/sys/test_pulp3_filesystem.py

Reply:
@swadeley Great! I think that's the perfect place for such a test. 

@latran With that test in place, is there anything actually specific to the content view API that you intend to test here to ensure pulp3 is present? Presence of specific fields in GET type requests, specific search/list parameters, POST parameters?